src/ol/style/index.jsdoc

```diff
@@ -28,16 +28,16 @@
  *  var white = [255, 255, 255, 1];
  *  var blue = [0, 153, 255, 1];
  *  var width = 3;
- *  styles[module:ol/geom/GeometryType~GeometryType.POLYGON] = [
+ *  styles[module:ol/geom/GeometryType.POLYGON] = [
```

Comment:
Seems like we should write valid JS here.  Either 'Polygon' or GeometryType.POLYGON, right?